.github/workflows/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.5.5
```

Comment:
Does it work?

Reply:
This is needed for Scala 2.13 support, I will remove this change in this PR.